components/Icons/Discord.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default (props) => (
+  <svg {...props} viewBox="0 0 300 300">
```

Comment:
make sure if you are spreading props onto a plain element that you aren't putting non-html tags on it since it will cause react to give warnings.